src/lang/en-US.ts

```diff
@@ -75,9 +75,10 @@ const CTAs: ResolvedIntlConfig['messages'] = {
     'cta.restart': `Restart`,
     'cta.enable': `Enable`,
     'cta.disable': `Disable`,
-    'cta.testConfig': `Test Catalog`,
-    'cta.generateCatalog': `Generate Catalog`,
-    'cta.regenerateCatalog': `Regenerate Catalog`,
+    'cta.testConfig': `Test`,
+    'cta.generateCatalog.capture': `Discover Endpoint`,
+    'cta.generateCatalog.materialization': `Validate Endpoint Config`,
```

Comment:
Generate buttons are custom for capture and materialization. Making them have their own content.